univocity-trader-core/src/main/java/com/univocity/trader/strategy/Indicators.java

```diff
@@ -255,6 +255,10 @@ public static MovingAverage MovingAverage(int length, TimeInterval interval, ToD
 		return register(new MovingAverage(length, interval, valueGetter), length, interval, valueGetter);
 	}
 
+	public static RealBodyIndicator RealBodyIndicator(TimeInterval interval) {
+		return register(new RealBodyIndicator(interval));
```

Comment:
note here that the "register" method should take the same parameters passed to the constructor.

This line should be written as: `return register(new RealBodyIndicator(interval), interval);`